frontend/tet/admin/package.json

```diff
@@ -32,7 +32,7 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-hook-form": "^7.30.0",
-    "react-leaflet": "^3.2.5",
+    "react-leaflet": "^4.0.0",
```

Comment:
Was there a dependency issue that required 4.0.0? I'm a bit worried about react-leaflet-marketcluster, since it hasn't really been supported lately. There wasn't really any good alternatives that would do the same (good clustering with markers) so had go with that. But luckily i has worked well lately. I tested with react-leaflet 4.0 and it seems to still work :)

Reply:
I can revert it, I already did from another package but didnt see the other one: https://github.com/City-of-Helsinki/yjdh/pull/1183/commits/2f014d302d1ce8866fe89ff93b435328fd1e2876
in tet/youth it caused a test to fail.

Reply:
Ok. Yeah, it might be safe to just keep react-leaflet on 3.x-version. Hope it doesn't clash with other updates in the future